lib/nostrum/api/adapter.ex

```diff
@@ -1,4 +1,4 @@
-defmodule Nostrum.Api.Base do
+defmodule Nostrum.Api.Adapter do
```

Comment:
Part of me wonders about renaming this module to `Nostrum.ApiAdapter` and similarly renaming the ratelimiter to `Nostrum.ApiAdapter.Ratelimiter` or something to that effect to have better separation of concerns.